### PR TITLE
Section Admin

### PR DESCRIPTION
**Pull request by heronakam**
This PR contains the work of both @G-boy05 and I on the admin section.

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
I dont know what you are doing, but your PR is totally broken. Simply look the 'changes' section here
And I thought Mohamed was already working on that ?

**Comment by heronakam** [comment]
Mohamed doesn't work on this admin section he worked just on the file configuration(admin.module and admin.route) on this folder

File: src/main/webapp/app/admin/admin.route.ts [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+    metricsRoute
+=======
 import { Routes, CanActivate } from '@angular/router';
 
 import {
     // configurationRoute,
     healthRoute,
     // logsRoute,
     // metricsRoute
+>>>>>>> refs/remotes/origin/angular-2
```
Comment (mohamed2a) [inline src/main/webapp/app/admin/admin.route.ts]: You need to remove the useless part of this change
+<<<<<<< HEAD
...
+=======
....
+>>>>>>> refs/remotes/origin/angular-2

**Comment by g-boy05** [comment]
We are correct the PR with @heronakam

**Comment by mohamed2a** [comment]
Related to #98

**Comment by jdubois** [comment]
Closing as this is done now

**Pull request closed without merging**